platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/eventhandling/TransactionPool.java

```diff
@@ -206,6 +207,14 @@ public synchronized boolean submitTransaction(
         return true;
     }
 
+    /**
+     * Same as {@link #submitTransaction(ConsensusTransactionImpl, boolean)} but with priority set to true.
+     * This method has no return since system transactions are never rejected.
+     */
+    public synchronized void submitSystemTransaction(@NonNull final SystemTransaction transaction) {
```

Comment:
missing param doc for `transaction`

Reply:
I left this out on purpose since I reference another method in the javadoc. I don't like repeating too much